crates/ursa-proxy/src/cache/tlrfu/mod.rs

```diff
@@ -0,0 +1,185 @@
+mod lru;
```

Comment:
Copypasta from gateway. Maybe this should be released as a crate so we can just wrap it up like moka. Thoughts?

Reply:
That's sounds like a good idea, I see this can be used for other purposes. I would cc @scott-dn 

Maybe not in this PR

Reply:
Sounds good